src/Sylius/Bundle/CoreBundle/Controller/ProductController.php

```diff
@@ -283,4 +332,51 @@ private function renderResults(
 
         return $this->handleView($view);
     }
+
+    /**
+     * @param ArchetypeInterface $archetype
+     * @param Pagerfanta $results
+     * @param string $template
+     * @param int $page
+     * @param null $facets
+     * @param null $facetTags
+     * @param null $filters
+     * @param null $searchTerm
+     * @param null $searchParam
+     * @param null $requestMethod
+     *
+     * @return Response
+     */
+    private function renderArchetypeResults(
+        ArchetypeInterface $archetype,
+        Pagerfanta $results,
+        $template,
+        $page,
+        $facets = null,
```

Comment:
I am good with leaving it, but can you tell me do you really need it? As I see you have never used this parameters, or I missed something? If not, can't we just delete them? If someone will need them, he can add them, but for now it seems to me that YAGNI...